geode-core/src/main/java/org/apache/geode/management/internal/cli/functions/DataCommandFunction.java

```diff
@@ -633,8 +631,8 @@ public DataCommandResult put(String key, String value, boolean putIfAbsent, Stri
       try {
         keyObject = getClassObject(key, keyClass);
       } catch (ClassNotFoundException e) {
-        return DataCommandResult.createPutResult(key, null, null,
-            "ClassNotFoundException " + keyClass, false);
+        return DataCommandResult.createPutResult(key, null, e.getException(),
```

Comment:
I don't understand the context of this change - the message was "ClassNotFound" but now it's "key not found" with a key class?